src/server/visual-diff-plugin.js

```diff
@@ -48,8 +60,10 @@ export function visualDiff() {
 
 			const browser = session.browser.name.toLowerCase();
 			const { dir, newName } = extractTestPartsFromName(payload.name);
-			const goldenFileName = `${join(dirname(session.testFile), 'screenshots', ciDir, 'golden', browser, dir, newName)}.png`;
-			const currentFileName = `${join(dirname(session.testFile), 'screenshots', ciDir, 'current', browser, dir, newName)}.png`;
+			const testPath = dirname(session.testFile).replace(rootDir, '');
+			const goldenFileName = `${join(rootDir, '.vdiff', testPath, 'golden', browser, dir, newName)}.png`;
+			const passFileName = `${join(rootDir, '.vdiff', testPath, 'pass', browser, dir, newName)}.png`;
+			const screenshotFileName = `${join(rootDir, '.vdiff', testPath, 'fail', browser, dir, newName)}.png`;
```

Comment:
This is an example of what this looks like:
![image](https://github.com/BrightspaceUI/testing/assets/13419300/f8243250-f376-438d-9c5f-8feab2f9e971)

Reply:
Haha yeah, I can call them something else to make it clearer, although I think this is a good example of the potential garbage pathing you can easily create 😛

Reply:
Ok, after meeting with Danny it became clear this was super confusing for no reason lol.  I've updated it, it now looks like this:
![image](https://github.com/BrightspaceUI/testing/assets/13419300/1a6a09c7-db62-4495-a33a-ae96590aface)
Still open to debate on whether we like this, but at least it's a bit clearer now what _this_ is.